docs/book/starter-guide/pipelines/dashboard.md

```diff
@@ -0,0 +1,76 @@
+---
+description: How to use ZenML's native dashboard to monitor your pipelines
+---
+
+# Dashboard 101
+
+ZenML comes with a native dashboard that allows you to monitor your pipelines
+and their runs. It is a great way to get a quick overview of your pipelines and
+their runs, and to inspect the artifacts that were produced by your pipeline
+runs.
+
+## How to use it
+
+The ZenML dashboard is served via the ZenML server. You must be connected to a
+local or deployed server in order to use the dashboard.
+
+The new ZenML Dashboard is now bundled into the ZenML Python package and can be
+launched directly from Python. The source code lives in the
+[ZenML Dashboard repository](https://github.com/zenml-io/zenml-dashboard).
+
+### Local dashboard
+
+To launch it locally, simply run `zenml up` on your machine and follow the
+instructions:
+
+```bash
+$ zenml up
+Deploying a local ZenML server with name 'local'.
+Connecting ZenML to the 'local' local ZenML server (http://127.0.0.1:8237).
+Updated the global store configuration.
+Connected ZenML to the 'local' local ZenML server (http://127.0.0.1:8237).
+The local ZenML dashboard is available at 'http://127.0.0.1:8237'. You can
+connect to it using the 'default' username and an empty password.
+```
+
+The Dashboard will be available at `http://localhost:8237` by
+default:
+
+![ZenML Dashboard Preview](../../assets/migration/zenml-dashboard.png)
+
+To start the dashboard locally, run the following command:
+
+```shell
+zenml up
```

Comment:
Feels like a repetition of the first sentence

Reply:
Removed it.